components/tidb_query/src/rpn_expr/impl_string.rs

```diff
@@ -39,6 +39,25 @@ pub fn concat(args: &[&Option<Bytes>]) -> Result<Option<Bytes>> {
     Ok(Some(output))
 }
 
+#[rpn_fn(varg, min_args = 2)]
+#[inline]
+pub fn concat_ws(args: &[&Option<Bytes>]) -> Result<Option<Bytes>> {
```

Comment:
According to TiDB's implementation I think this function accepts multiple parameters?

Reply:
I try `select concat_ws('a)` on MySQL and it throw an incorrect params count error to me.